### ISSUE TITLE
Some questions about the paper.

### ISSUE THREAD

**Issue by duzhenjiang113** [Mathux/ACTOR#3]
Hello, it's a great work. But I have a question about this paper. This paper gave me a lot of inspiration. For the Decoder, directly generating all the results at one time will make the training process difficult. So what method is used in the paper to ensure that the decoder is provided with a inductive bias to ensure that it can perform well during training?

**Comment by Mathux** [comment]
Hello
> For the Decoder, directly generating all the results at one time will make the training process difficult.

I don't think it makes the learning process difficult. I use AdamW with a fixed learning rate without any special tricks.

> So what method is used in the paper to ensure that the decoder is provided with a inductive bias to ensure that it can perform well during training?

The decoder uses the latent vector shifted by the action bias term, and positional encodings. No particular method is used to ensure anything during training.
